00_Base/src/interfaces/cache/cache.ts

```diff
@@ -77,6 +77,7 @@ export interface ICache {
    * */
   setIfNotExist(key: string, value: string, namespace?: string, expireSeconds?: number): Promise<boolean>;
 
+  // TODO: Consider removing this method, no longer used
```

Comment:
I had the same back-and-forth in my mind as you did. Originally thought I should just remove it if it wasn't in use, then thought it may still belong because it is useful to have as an option. I just wanted other opinions. If you agree it should stay, I'll remove the TODO!

Reply:
Yea, I think it's fine to keep it for now. It does make it more complete.